### PR TITLE
Feature/15 speed units

### LINKED ISSUE #11: Jupyter Notebook File Failing
When the jupyter notebook file tried to run (**_/examples/notebooks/data_store_sqlite.ipynb_**) , it throws the following error:
```
OperationalError: (sqlite3.OperationalError) no such table: Datafiles
[SQL: SELECT "Datafiles".datafile_id AS "Datafiles_datafile_id", "Datafiles".simulated AS "Datafiles_simulated", "Datafiles".reference AS "Datafiles_reference", "Datafiles".url AS "Datafiles_url", "Datafiles".privacy_id AS "Datafiles_privacy_id", "Datafiles".datafile_type_id AS "Datafiles_datafile_type_id" 
FROM "Datafiles" 
WHERE "Datafiles".reference = ?
 LIMIT ? OFFSET ?]
[parameters: ('../repl_files/missing_platform.rep', 1, 0)]
```
Screenshot from 2019-11-07 13-33-51

We should examine and fix it.

### LINKED ISSUE #15: Speed Unit Failing
When a new object tried to push to **State** table, _speed_ attribute causes the following error:

Screenshot from 2019-11-07 15-27-23

Screenshot from 2019-11-07 15-29-21

### PR DESCRIPTION
**Pull request by IanMayo**
Fixes #15 
Fixes #11 
It also starts to delve down into Command Line Resolver.  It's all quite new to me.  The existing version always collected data necessary for referential integrity.  

But, sometimes that data is already available - so we don't need to.

### CODE REVIEW

**Comment by IanMayo** [comment]
@BarisSari - as before, I don't mind the code coverage dropping - that's ok while we're still finding our feet.

File: examples/notebooks/data_store_sqlite.ipynb [hunk 1]
```diff
@@ -20,7 +22,7 @@
     "import os\n",
     "import sys\n",
     "\n",
-    "module_path = os.path.abspath(os.path.join('..'))\n",
+    "module_path = os.path.abspath(os.path.join('../../'))\n",
```
Comment (BarisSari) [inline examples/notebooks/data_store_sqlite.ipynb]: Actually, I haven't checked this cell after migrating to the new repository. Have you tried to run the notebook without it? Was it necessary to add module paths in `../..` ?

File: pepys_import/core/debug/support_methods.py [hunk 2]
```diff
@@ -0,0 +1,123 @@
+from sqlalchemy.orm import sessionmaker
+from tabulate import tabulate
+
+from pepys_import.core.store.sqlite_db import State, Platform, Sensor, Nationality, PlatformType
+
+class Support_Methods():
```
Comment (BarisSari) [inline pepys_import/core/debug/support_methods.py]:
```suggestion
class SupportMethods():
```

File: tests/test_data_store_initialise.py [hunk 3]
```diff
@@ -14,8 +14,7 @@ def setUp(self):
                 database="test", host="localhost", user="postgres", port=55527
             )
         except RuntimeError:
-            print("PostgreSQL database couldn't be created! "
-                  "Test is skipping.")
+            print("PostgreSQL database couldn't be created! " "Test is skipping.")
```
Comment (BarisSari) [inline tests/test_data_store_initialise.py]:
```suggestion
            print("PostgreSQL database couldn't be created! Test is skipping.")
```

Comment (BarisSari) [reply tests/test_data_store_initialise.py]: I think it is created by Black.

File: examples/notebooks/data_store_sqlite.ipynb (hunk 1 above)
Comment (IanMayo) [reply examples/notebooks/data_store_sqlite.ipynb]: I needed to do this to make it work.  The current file is an extra level down.

File: pepys_import/core/debug/support_methods.py (hunk 2 above)
Comment (IanMayo) [reply pepys_import/core/debug/support_methods.py]: Done in new commit.

**Comment by IanMayo** [comment]
Ok, I've finished for the night. No more changes, honest!

File: examples/notebooks/data_store_sqlite.ipynb (hunk 1 above)
Comment (BarisSari) [reply examples/notebooks/data_store_sqlite.ipynb]: Okay, PyCharm worked without it and therefore I wanted to ask. It is probably setting up the project environment by default.

**Comment by BarisSari** [comment]
> Ok, I've finished for the night. No more changes, honest!

Good work, I think you can merge the branch! By the way, it fixes #11 as well.

**Pull request merged**